src/reference/common_errors.md

```diff
@@ -80,6 +80,38 @@ Time synchronization instructions for Linux can be found [here](https://unix.sta
 1. Once the installation is complete, the NTP service should start automatically.
 1. Check its status by using this command `sudo systemctl status ntp`.
 
+## Error: Unexpected error when rendering on the frontend.
+
+**If you encounter this error, you are running Caido as root with `--no-renderer-sandbox`. In the [logs](/reference/configuration/data_location.md), you will see a similar message to the following:**
+
+```
+Rendering error: LaunchIo(Custom { kind: UnexpectedEof, error: "unexpected end of stream" }, BrowserStderr("[0101/110718.156035:ERROR:zygote_host_impl_linux.cc(90)] Running as root without --no-sandbox is not supported. See https://crbug.com/638180.\n"))
+```
+
+### Resolution:
+
+Don't run Caido as root and remove the flag `--no-renderer-sandbox` when launching Caido.
+
+## Error: Could not initialize configuration.
+
+**If you encounter this error and receive a 404 status code on `https://api.caido.io/oauth2/register` along with a similar message to the following:**
+
+```
+Error: Could not initialize configuration
+Caused by:
+    0: Authentication service error
+    1: Cloud operation failed
+    2: Cloud unavailable
+    3: error sending request for url (https://api.caido.io/oauth2/register): error trying to connect: tcp connect error: Connection refused (os error 111)
+    4: error trying to connect: tcp connect error: Connection refused (os error 111)
+    5: tcp connect error: Connection refused (os error 111)
+    6: Connection refused (os error 111)
+```
+
+### Resolution:
+
+Check your internet connection.
```

Comment:
We should likely explain the following:
Caido requires an internet connection at a couple critical moments:
  - On first launch
  - During login
  - After 7 days offline
We can link to the authentication documentation too.